rsts/deployment/cluster_config/general.rst

```diff
@@ -103,21 +101,21 @@ Then run the below command to ensure that when the admin fills in cluster resour
 Refer to the :ref:`docs <flytectl:flytectl_update_cluster-resource-attribute>` to learn more about the command and its supported flag(s).
 
 
-To fetch and verify the individual project-domain attributes, similarly you can run:
+To fetch and verify the individual project-domain attributes:
 
 .. prompt:: bash
 
     flytectl get cluster-resource-attribute -p flyteexamples -d development
 
 Refer to the :ref:`docs <flytectl:flytectl_get_task-resource-attribute>` to learn more about the command and its supported flag(s).
 
-The above-updated values will, in turn, be used to fill in the template fields for the flyteexamples-development namespace.
+You can use these updated values to fill the template fields for the flyteexamples-development namespace.
 
 For other namespaces, the `platform defaults <https://github.com/flyteorg/flyte/blob/1e3d515550cb338c2edb3919d79c6fa1f0da5a19/charts/flyte-core/values.yaml#L719,L734>`__ apply.
 
 .. note::
-    The template values, e.g. ``projectQuotaCpu`` or ``projectQuotaMemory`` are freeform strings.
-    You must ensure that they match the template placeholders in your `template file <https://github.com/flyteorg/flyte/blob/master/kustomize/base/single_cluster/headless/config/clusterresource-templates/ab_project-resource-quota.yaml>`__
+    The template values, for example: ``projectQuotaCpu`` or ``projectQuotaMemory`` are free-form strings.
```

Comment:
```suggestion
    The template values, for example, ``projectQuotaCpu`` or ``projectQuotaMemory`` are free-form strings.
```
I don't think you should use a colon (`:`) when using "for example" in a sentence.